internal/controller/replication.storage/volumereplication_controller.go

```diff
@@ -160,7 +160,7 @@ func (r *VolumeReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		pvc, pv, pvErr = r.getPVCDataSource(logger, nameSpacedName)
 		if pvErr != nil {
 			logger.Error(pvErr, "failed to get PVC", "PVCName", instance.Spec.DataSource.Name)
-			setFailureCondition(instance)
+			setFailureCondition(instance, "failed to get PVC")
```

Comment:
failed to find the PVC? "Get" is a technical term, the function used to read an object from the api server, the human term is finding or looking up.

But it is not clear if the issue is missing pvc (ErrNotExist) or an error accessing the api server (maybe temporary error?). When we fix the code to handle both case we can provide a more useful error message.